src/idom/core/hooks.py

```diff
@@ -281,7 +277,7 @@ def __init__(
 
     def render(self) -> VdomDict:
         current_hook().set_context_provider(self)
-        return vdom("", *self.children)
+        return {"tagName": "", "children": self.children}
```

Comment:
Is there a reason this was switch to a raw VDOM dict?

Reply:
Just faster. No particular reason.